src/createQueryGroup.ts

```diff
@@ -171,19 +171,30 @@ export function createQueryGroup<
       .from(subscriptions.values())
       .map(subscription => subscription.retries)
     
-    retries.push(options?.retries)
+    retries.push(groupOptions?.retries)
 
     return reduceRetryOptions(retries)
   }
 
   function subscribe<
     TOptions extends QueryOptions<TAction>
   >(options?: TOptions): Query<TAction, TOptions> {
-    const removeSubscription = addSubscription(options)
+    const mergedOptions = {
+      ...groupOptions,
+      ...options,
+    }
```

Comment:
Not sure how i feel about this, I don't think merging the query options with the group options makes sense. Should we just call the `onDispose` callback of the group separately from the on dispose callback of the individual queries/subscription?